src/services/legal-services.ts

```diff
@@ -286,7 +288,7 @@ export default class LegalServices {
         return data
       }).catch(error => {
         if (error?.response?.status === StatusCodes.NOT_FOUND) {
-          return null // Business or Address not found (not an error)
+          return {} // Business or Address not found (not an error)
```

Comment:
Should this return `{ businessOffice: null }` as in Business Dashboard?

Ref: https://github.com/bcgov/business-dashboard-ui/blob/6fcb95388d2ab122a73bb5e4af6ac38bfb512389/src/stores/business.ts#L128